modules/nf-core/goleft/indexcov/main.nf

```diff
@@ -0,0 +1,54 @@
+
+process GOLEFT_INDEXCOV {
+    tag 'N=${bams.size()}'
+    label 'process_single'
+
+    conda "bioconda::goleft=0.2.4"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/goleft:0.2.4--h9ee0642_1':
+        'biocontainers/goleft:0.2.4--h9ee0642_1' }"
+
+    input:
+    tuple val(meta),path(bams),path(indexes)
+    path  fai
```

Comment:
the fai should also get a `meta2` . We started adding it to all modules, mostly coming from the metagenomics corner where they have multiple references sometimes and having it consistent makes channel logic so much easier